README.md

```diff
@@ -1,32 +1,32 @@
-#Wiznet_5500
+# Wiznet W5500
 
-Connection API for Wiznet chip [W5500](http://wizwiki.net/wiki/lib/exe/fetch.php?media=products:w5500:w5500_ds_v106e_141230.pdf).  The W5500 chip is a hardwired TCP/IP embedded Ethernet controller that enables easier Internet connection for embedded systems.  This library supports SPI integration with the W5500.
+This library allows you to communicate with a TCP/IP network (separate from the Imp's connection to the network) using the Wiznet W5500 chip [W5500](http://wizwiki.net/wiki/lib/exe/fetch.php?media=products:w5500:w5500_ds_v106e_141230.pdf). The W5500 chip is a hardwired TCP/IP embedded Ethernet controller. This library supports SPI integration with the W5500.
 
-**To add this code to your project, copy and paste the entire contents of the** `W5500.class.nut` **file just below your library *#require* statements at the top of your device code.**
+**To use this library, add `#require "W5500.device.nut:1.0.0"` to the top of your device code.**
 
-## Class Usage
+## Class W5500
 
-### Constructor: W5500.API(*spi, interruptPin[, csPin][, resetPin]*)
+### Constructor: W5500(*interruptPin, spi[, csPin], resetPin[, autoRetry]*)
```

Comment:
seems odd to have a required parameter in the middle of some optional parameters.